packages/a2/src/a2/utils.ts

```diff
@@ -6,6 +6,8 @@ import {
   Outcome,
   StoredDataCapabilityPart,
 } from "@breadboard-ai/types";
+import * as BreadboardUiTypes from "@breadboard-ai/shared-ui/types";
```

Comment:
This creates a circular reference. shared-ui depends on a2 and now a2 depends on shared ui